### ISSUE TITLE
Per user disable authentication not working

### ISSUE THREAD

**Issue by clarknelson** [born05/craft-twofactorauthentication#81]
I have a website where each user shows that two-factor auth is disabled, even if they actually are enabled (in the database, practically). It seems like the issue is resolved temporarily when you clear the Craft CMS cache. Not sure how else to debug from here.

Thanks,
– Clark

**Comment by clarknelson** [comment]
Any ideas what might be causing this issue? I'm going to pick through the code this week to see why the user's 2fa reset button isn't showing up.

**Comment by roelvanhintum** [comment]
Hi @clarknelson, do you mean the "2-Factor Auth" column in the user overview in Craft's CP? I can't reproduce this myself. Can you give some info on Craft and plugin versions?

**Comment by clarknelson** [comment]
Thanks for taking a look. Yes for a user this "2FA disable" button does not appear: 

<img width="371" alt="CleanShot 2023-08-16 at 09 43 47@2x" src="https://github.com/born05/craft-twofactorauthentication/assets/4106671/364f1b7d-9888-4a0a-9fae-8f716b1a07bc">

I believe it works on the live site running these plugin versions:
```
"craftcms/cms": "4.3.6.1",
"born05/craft-twofactorauthentication": "3.1.0",
```

But not on the staging website running these plugin versions:
```
"craftcms/cms": "4.3.6.1",
"born05/craft-twofactorauthentication": "3.2.1",
```

It appeared for me after clearing the cache, but this user does not have access to the "clear cache" button.

Thanks,
– Clark

**Comment by roelvanhintum** [comment]
Is this fixed with ~~3.3.0~~?

Edit: 3.3.1

**Comment by clarknelson** [comment]
Sorry for the delay, I think the issue was on our end... I believe the user's permissions to access the plugin were removed somehow.
